takwimu/templates/takwimu/_includes/navbar/about.html

```diff
@@ -5,68 +5,65 @@
 
 {% block dropdown_id %}nav-about{% endblock %}
 
-{% block dropdown_title %}About Takwimu{% endblock %}
-{% block dropdown_desc %}Takwimu provides quality insights and data-driven analysis to empower African development champions.{% endblock %}
+{% block dropdown_title %}<p class="text-right pr-3">About Takwimu</p>{% endblock %}
+{% block dropdown_desc %}<p class="text-right pr-3 font-weight-light">Takwimu provides quality insights and data-driven analysis to empower African development champions.</p>{% endblock %}
 
 {% block dropdown_content %}
 <div class="row pl-4 pb-5" style="width: 100%;">
   <div class="col-sm-4">
     <ul style="color:#7ab1f2; font-weight: bold;">
       <li class="pb-2" type="disc"><a href="{% url 'about_page' %}"><u>About Takwimu</u></a></li>
-      <li class="pb-2" type="disc"><a href="/about/support-services"><u>Support Services</u></a></li>
+      <li class="pb-2" type="disc"><a href="/about/support-services"><u>Our Data Services</u></a></li>
+      <li class="pb-2" type="disc"><a href="https://medium.com/takwimu-africa"><u>Insights &amp; Stories</u></a></li>
       <li class="pb-2" type="disc"><a href="/contact-us"><u>Contact Us</u></a></li>
       <li class="mb-0 pb-3" type="disc"><a href="/dashboard"><u>Dashboard (Admin)</u></a></li>
     </ul>
   </div>
-  <div class="col-sm-4">
-    <h5 class="pb-2">Follow us</h5>
+  <div class="col-sm-8">
+    <p class="font-weight-bold pb-0">Follow us</p>
```

Comment:
Please use heading tag (`h*`) instead of paragraph tag for these (`p`) because they are headings and not body text.